src/System.Diagnostics.Debug/src/System/Diagnostics/DebuggerVisualizerAttribute.cs

```diff
@@ -11,36 +11,40 @@ namespace System.Diagnostics
     [AttributeUsage(AttributeTargets.Struct | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]
     public sealed class DebuggerVisualizerAttribute : Attribute
     {
-        private readonly string _visualizerObjectSourceName;
-        private readonly string _visualizerName;
         private Type _target;
 
         public DebuggerVisualizerAttribute(string visualizerTypeName)
         {
-            _visualizerName = visualizerTypeName;
+            VisualizerTypeName = visualizerTypeName;
         }
+
         public DebuggerVisualizerAttribute(string visualizerTypeName, string visualizerObjectSourceTypeName)
         {
-            _visualizerName = visualizerTypeName;
-            _visualizerObjectSourceName = visualizerObjectSourceTypeName;
+            VisualizerTypeName = visualizerTypeName;
```

Comment:
I assume you only made this public so that we can run tests for them. You will only have access to them through the tests though because they compile against the runtime, but the reference assembly will still not expose them.